test/contract-tests/images/applications/TestSimpleApp.AWSSDK.Core/BedrockTests.cs

```diff
@@ -0,0 +1,149 @@
+using System.Net;
+using System.Text;
+using System.Text.Json;
+using Amazon.Bedrock;
+using Amazon.Bedrock.Model;
+using Amazon.BedrockRuntime;
+using Amazon.BedrockRuntime.Model;
+using Amazon.BedrockAgent;
+using Amazon.BedrockAgent.Model;
+using Amazon.BedrockAgentRuntime;
+using Amazon.BedrockAgentRuntime.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace TestSimpleApp.AWSSDK.Core;
+
+public class BedrockTests(
+    IAmazonBedrock bedrock,
+    IAmazonBedrockRuntime bedrockRuntime,
+    IAmazonBedrockAgent bedrockAgent,
+    IAmazonBedrockAgentRuntime bedrockAgentRuntime,
+    ILogger<BedrockTests> logger) :
+    ContractTest(logger)
+{
+    public Task<GetGuardrailResponse> GetGuardrail()
+    {
+        return bedrock.GetGuardrailAsync(new GetGuardrailRequest
+        {
+            GuardrailIdentifier = "test-guardrail",
+        });
+    }
+
+    public GetGuardrailResponse GetGuardrailResponse()
+    {
+        return new GetGuardrailResponse
+        {
+            HttpStatusCode = HttpStatusCode.OK,
+            GuardrailId = "test-guardrail",
+        };
+    }
+
+    public void InvokeModel()
+    {
+        bedrockRuntime.InvokeModelAsync(new InvokeModelRequest
+        {
+            ModelId = "test-model",
+        });
+        return;
+    }
+
+    public void InvokeModelResponse()
+    {
+        return;
```

Comment:
Why does the span still have http.status_code attributes even we don't return it in the response?

Reply:
For .NET framework, it returns 200 OK if the method completes without exceptions.